modes/debug_mode/debug_table_page.py

```diff
@@ -177,7 +177,7 @@ def enter_button_pressed(self):
             self.table[self.selected_id].highlight("gray")
             self.model.remove_pinned_data(self.selected_id)
         # otherwise if the selected id is not pinned and there are less than 6 pinned then pin it
-        elif len(self.model.pinned_data.keys()) < 6:
+        elif len(self.model.pinned_data) < 6:
```

Comment:
6 should probably be a constant like MAX_ITEMS_PINNED or whatever you want to call it